src/stores/modules/Video.store.js

```diff
@@ -29,6 +29,9 @@ const getters = {
         tagItem.who = "Cacophony AI";
         tagItem['_rowVariant'] = 'warning';
       } else {
+        if (!tag.tagger) {
+          return;
```

Comment:
Is it best to show the tag without an tagger?

Reply:
I agree that the tag should still be shown. Just show the tagger as an empty string or "-".